perf_test/blas/blas3/KokkosBlas3_trmm_perf_test.hpp

```diff
@@ -626,25 +632,30 @@ void __do_loop_and_invoke(options_t options, void (*fn)(options_t, trmm_args_t))
 /*************************** External fns **************************/
 void do_trmm_serial_blas(options_t options) {
   STATUS;
-  __do_loop_and_invoke(options, __do_trmm_serial_blas<default_scalar, view_type_3d, view_type_3d, default_device>);
+  __do_loop_and_invoke(
```

Comment:
Not for this PR but these leading double underscores are really ill advised...

Reply:
I will fix it in a future one 😄